src/Sarif.FunctionalTests/SarifConverterTests.cs

```diff
@@ -88,12 +91,40 @@ private void BatchRunConverter(string tool, string inputFilter = "*.xml")
 
             foreach (string file in testFiles)
             {
-                RunConverter(sb, tool, file);
+                string actualFilePath = RunConverter(sb, tool, file);
+
+                if (enrichConvertedSarif && File.Exists(actualFilePath))
+                {
+                    EnrichSarifLog(actualFilePath);
+                }
             }
 
             sb.Length.Should().Be(0, FormatFailureReason(sb, tool));
         }
 
+        private static void EnrichSarifLog(string actualFilePath)
+        {
+            JsonSerializerSettings settings = new JsonSerializerSettings()
+            {
+                Formatting = Formatting.Indented,
+                ContractResolver = SarifContractResolver.Instance
+            };
+
+            string logText = File.ReadAllText(actualFilePath);
+
+            SarifLog actualLog = JsonConvert.DeserializeObject<SarifLog>(logText, settings);
+
+            OptionallyEmittedData dataToInsert =
```

Comment:
>OptionallyEmittedData [](start = 12, length = 21)

Consider a comment: "If you change the set of data to insert, make the corresponding change to the invocation of Sarif.Multitool in UpdateBaselines.ps1"

Reply:
Note to self: _are_ those two things related?

---
In reply to: [212415835](https://github.com/Microsoft/sarif-sdk/pull/1044#discussion_r212415835) [](ancestors = 212415835)